docs/source/llms/llm-evaluate/index.rst

```diff
@@ -494,6 +497,178 @@ up OpenAI authentication to run the code below.
+
+Please don't forget to set the target deployment client by using :py:func:`mlflow.deployments.set_deployments_target` before calling :py:func:`mlflow.evaluate()` with the endpoint URI, as shown in the example below. Otherwise, you will see an error message like ``MlflowException: No deployments target has been set...``.
+
+.. hint::
+
+    When you want to use an external endpoint **not** hosted by an MLflow Deployments Server or Databricks, you can create a custom Python function following the :ref:`Evaluating with a Custom Function <llm-eval-custom-function>` guide and use it as the ``model`` argument.
+
+Supported Input Data Formats
+****************************
+
+The input data can be either of the following format when using an URI of the MLflow Deployment Endpoint as the model:
+
+.. list-table::
+    :widths: 20 40 40
+    :header-rows: 1
+    :class: wrap-table
+
+    * - Data Format
+      - Example
+      - Additional Notes
+
+    * - A pandas DataFrame with a string column.
+      - 
+        .. code-block:: python
+
+            {
+                "inputs": [
+                    "What is MLflow?",
+                    "What is Spark?",
+                ]
+            }
+
+      - For this input format, MLflow will construct the appropriate request payload to the model endpoint type. For example, if your model is a chat endpoint (``llm/v1/chat``), MLflow will wrap your input string with the chat messages format like ``{"messages": [{"role": "user", "content": "What is MLflow?"}]}``. If you want to customize the request payload e.g. including system prompt, please use the next format.
+
+    * - A pandas DataFrame with a dictionary column.
+      - 
+        .. code-block:: python
+
+            {
+                "inputs": [
+                    {
+                        "messages": [
+                            {"role": "system", "content": "Please answer."},
+                            {"role": "user", "content": "What is MLflow?"},
+                        ],
+                        "max_tokens": 100,
+                    },
+                    # ... more dictionary records
+                ]
+            }
+
+      - In this format, the dictionary should have the correct request format for your model endpoint.
+
+    * - A (nested) list of input strings.
+      - 
+        .. code-block:: python
+
+            [
+                ["What is MLflow?"],
+                ["What is Spark?"],
+            ]
+
+      - The :py:func:`mlflow.evaluate()` will also accepts a list input. One notable requirement is that the each
+        list element i.e. input string needs to be wrapped in another list, so they can be passed as a single prediction request to the model endpoint.
+
+    * - A (nested) list of input strings.
+      - 
+        .. code-block:: python
+
+            [
+                [
+                    {
+                        "messages": [
```

Comment:
Can we bit a bit more specific in the description about what that dictionary column should contain - i.e. the `messages` key for the MLflow chat completions format (would also be great to link to docs for that if we have them)? I don't think it should contain a `max_tokens` key, just the `messages` key for now.

Reply:
Will add link to https://mlflow.org/docs/latest/llms/deployments/index.html#standard-query-parameters :)